fix(curriculum): Extended description (#47169)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/61537c9eecea6a335db6da79.md

# --description--

> Below your `header`, create a new `div` element and assign a class of `gallery` to it. The class of 'gallery' is an identifer that will be referenced to style the image gallery and make it responsive.

Reviewer: I think simpler is better:
```suggestion
Below your `header` element, create a new `div` element with the `id` of `"gallery"`. This `div` will act as a container for the gallery images.
```

Reviewer 2: I am worried about `"gallery"`, people may type `<div id=""gallery"">`, what do you think of removing the quotes? @jeremyIt

Reviewer: We have done ""s around string literals in other challenges. We should pick one and stick to it.
